libs/auth/src/dipped_smart_card_auth.ts

```diff
@@ -434,11 +439,16 @@ export class DippedSmartCardAuth implements DippedSmartCardAuthApi {
                     user.role === 'system_administrator' ||
                       user.role === 'election_manager'
                   );
-                  return isFeatureFlagEnabled(
+                  const skipPinEntry = isFeatureFlagEnabled(
                     BooleanEnvironmentVariableName.SKIP_PIN_ENTRY
```

Comment:
my feature flag baby is still alive, what great news!

Reply:
It's made it through many a refactor haha